backend/src/schema/resolvers/posts.js

```diff
@@ -131,10 +145,10 @@ export default {
         await session.run(cypherDeletePreviousRelations, { params })
 
         updatePostCypher += `
-          WITH post
           UNWIND $categoryIds AS categoryId
           MATCH (category:Category {id: categoryId})
           MERGE (post)-[:CATEGORIZED]->(category)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-17T14:59:29Z" title="Thursday, October 17th 2019, 4:59:29 pm +02:00">Oct 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -111,14 +128,17 @@ export default {
       return post
     },
     UpdatePost: async (_parent, params, context, _resolveInfo) => {
-      const { categoryIds } = params
```


I would have preferred to have separate graphql mutations `pin` and `unpin` for the post.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-17T22:46:22Z" title="Friday, October 18th 2019, 12:46:22 am +02:00">Oct 18, 2019</time>_

---


done, thanks for the reminder, I think it's much cleaner